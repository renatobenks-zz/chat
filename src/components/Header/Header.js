// @flow

import styled from 'styled-components';

const Header = styled.header`
  display: flex;
  flex: 1;
  justify-content: flex-start;
  align-items: center;
  background-color: ${props => props.theme.palette.white};
  border-bottom: 1.5px solid ${props => props.theme.palette.secondary};
  height: 90px;
  padding: ${props => props.theme.padding};
`;

export default Header;
