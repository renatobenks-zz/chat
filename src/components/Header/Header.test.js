import React from 'react';

import { render } from '../../../test/helpers';

import Header from './Header';

const renderer = component => render(component, { withTheme: true });

it('renders correctly', () => {
  expect(
    renderer(
      <Header>
        <span>text into header</span>
      </Header>,
    ),
  ).toMatchSnapshot();
});
