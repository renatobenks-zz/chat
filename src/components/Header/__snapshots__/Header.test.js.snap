// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<header
  className="sc-bdVaJa ghYoLl"
>
  <span>
    text into header
  </span>
</header>
`;
